docs/pages/Nakama/plugin.mdx

```diff
@@ -0,0 +1,201 @@
+# Cardinal Plugin
+
+Go based Nakama containers can be extended using shared objects (aka [plugins](https://pkg.go.dev/plugin)). The [starter-game-template](https://github.com/Argus-Labs/starter-game-template) leverages this shared object pattern to expand on Nakama to better interact with a Cardinal based backend.
+
+The Nakama plugin bundled with the starter-game-template is general purpose, and should work with your Cardinal based game without any modifications. As you work on your game, you may find that you want to further change the Nakama plugin to better serve your specific use case. This document serves as a reference for the various aspects of the Cardinal-focused Nakama plugin.
+
+## RPC Endpoints
+
+Transactions and Reads registered with a `cardinal.World` object will automatically get Nakama RPC endpoints created for them. For example, if you set up a Transaction and a Read with the following code:
+
+```go
+var MoveTx = cardinal.NewTransactionType[MoveMsg, MoveResult]("move-a-player")
+var PositionRead = cardinal.NewReadType[PositionRequest, PositionReply]("position")
+```
+
+And register those objects with `world.RegisterTransactions` and `world.RegisterReads`, the following RPC endpoints will be created in Nakama:
+
+- /tx/game/move-a-player
+- /read/game/position
+
+RPC requests to those Nakama endpoints will be passed along to your Cardinal implementation. Transactions will be added to the transaction queue (which can be processed at the next game tick in a registered System), and Reads will be passed directly to Cardinal.
+
+The communication from a client (e.g. the Nakama console or a custom-built game client) to Nakama uses the RPC protocol. Communication between Nakama and Cardinal makes use of HTTP. Request payloads and response payloads are JSON encoded strings.
+
+## Transaction Signatures
+
+When submitting transactions from Nakama to Cardinal, transactions must be cryptographically signed. The Cardinal plugin will use an existing private key, or generate one if one does not exist. This key generation and signing happens automatically. To make use of this automatic signing, a client must first register a >>>Persona Tag<<<(LINK_TO_MARKDOWN). Once a Persona Tag has been requested and accepted, future transactions will be automatically signed.
```

Comment:
Seems like the >>> <<< thing is causing syntax error, and i think you forgot to add the link to the markdown here

Reply:
I wasn't sure on the syntax. Hopefully these are fixed now.